docs/actors/development/input_schema.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Input schema
-description: Documentation of Apify actors - defining your actor's input schema.
+description: Learn how to define and structure your actor's input in the codebase or using a visual editor. Define input type and minimum/maximum values.
```

Comment:
Maybe something more like: Learn how to define schema for your actor's input to easily validate it it and to provide autogenerated input UI to its users.